db/migrate/20150330221154_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :body
+      t.integer :rating
+      t.references :restaurant, index: true, foreign_key: true
```

Comment:
these should also be constrained. You can't have a review without a user or a restaurant